test/tools/ossfuzz/config/strict_assembly.dict

```diff
@@ -0,0 +1,118 @@
+" -> "
+" := "
+"("
+")"
+", "
+":"
```

Comment:
Probably better to leave that out - it is only required for types.

Reply:
not sure I follow. The following Solidity assembly snippet from [here](https://solidity.readthedocs.io/en/v0.5.3/assembly.html) contains all of cited tokens

```
        assembly {
            // retrieve the size of the code, this needs assembly
            let size := extcodesize(_addr)
            // allocate output byte array - this could also be done without assembly
            // by using o_code = new bytes(size)
            o_code := mload(0x40)
            // new "memory end" including padding
            mstore(0x40, add(o_code, and(add(add(size, 0x20), 0x1f), not(0x1f))))
            // store length in memory
            mstore(o_code, size)
            // actually retrieve the code, this needs assembly
            extcodecopy(_addr, add(o_code, 0x20), 0, size)
```

Reply:
sure, but `:` only appears as part of `:=`.